### PR TITLE
LPS-111104

### PR DESCRIPTION
**Pull request by yuhai**

### CODE REVIEW

**Comment by yuhai** [comment]
ci:test:sf

**Comment by yuhai** [comment]
ci:test:relevant

File: modules/apps/portal/portal-log4j-test/src/test/java/com/liferay/portal/log4j/Log4jTest.java [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+		Date logDate = simpleDateFormat.parse(date);
+
+		StringBundler sb = new StringBundler(18);
+
+		sb.append("<log4j:event logger=\"");
+		sb.append(Log4jTest.class.getName());
+		sb.append("\" timestamp=\"");
+		sb.append(logDate.getTime());
+		sb.append("\" level=\"");
+		sb.append(Level.INFO);
```
Comment (dantewang) [inline modules/apps/portal/portal-log4j-test/src/test/java/com/liferay/portal/log4j/Log4jTest.java:381]: This seems to be the only usage of `log4j.Level` right? Could you replace it into a string?

File: modules/apps/portal/portal-log4j-test/src/test/java/com/liferay/portal/log4j/Log4jTest.java [hunk 2]
```diff
@@ -0,0 +1,466 @@
...
+
+	@AfterClass
+	public static void tearDownClass() {
+		_printStream.flush();
+
+		_printStream.close();
+
+		System.setOut(_originalOutputStream);
+
+		LogManager.shutdown();
```
Comment (dantewang) [inline modules/apps/portal/portal-log4j-test/src/test/java/com/liferay/portal/log4j/Log4jTest.java:123]: Is there any specific reason that we need this?

**Comment by dantewang** [comment]
@yuhai Please see my comments, and also fix the pmd violation. You should use `assertTrue` in your case. Also, in some places you `assertTrue(a.equals(b))`. That should be replaced by `assertEquals`.

**Pull request closed without merging**